trojsten/top30/tests.py

```diff
@@ -123,13 +117,47 @@ def test_most_submits_reviewed(self):
             leaderboards,
         )
 
+        response = self.client.get(self.url, {"all_sites": True})
+        self.assertEqual(response.status_code, 200)
+        leaderboards = [leaderboard for name, leaderboard in response.context["leaderboards"]]
+        self.assertIn(
+            [(2.5, self.users[2].get_full_name()), (1.5, self.users[3].get_full_name())],
+            leaderboards,
+        )
+
+    def test_most_submissions(self):
+        self._create_submit(self.task1, self.users[0], points=1)
+        self._create_submit(self.task2, self.users[0], points=1)
+        self._create_submit(self.task1, self.users[1], points=1)
+        self._create_submit(self.task1, self.users[1], points=1)
+        self._create_submit(self.task1, self.users[1], points=1, type=SUBMIT_TYPE_TEXT)
+        self._create_submit(self.task2, self.users[1], points=0)
+        self._create_submit(self.other_task1, self.users[0], points=4)
+
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 200)
+        leaderboards = [leaderboard for name, leaderboard in response.context["leaderboards"]]
+        self.assertIn(
+            [(2, self.users[0].get_full_name()), (1, self.users[1].get_full_name())],
+            leaderboards,
+        )
+
+        response = self.client.get(self.url, {"all_sites": True})
+        self.assertEqual(response.status_code, 200)
+        leaderboards = [leaderboard for name, leaderboard in response.context["leaderboards"]]
+        self.assertIn(
+            [(3, self.users[0].get_full_name()), (1, self.users[1].get_full_name())],
+            leaderboards,
+        )
+
     def _create_camps(self):
         place = EventPlace.objects.create(name="Camp place")
         type_camp = EventType.objects.create(name="Camp", organizers_group=self.group, is_camp=True)
         type_camp.sites.add(self.site)
         type_other = EventType.objects.create(
             name="Camp", organizers_group=self.group, is_camp=True
         )
+        type_other.sites.add(self.other_site)
```

Comment:
```suggestion
        type_other.sites.add(self.other_site)
        type_other.save()
```
Tiez ako pri minulych komentoch.